frontend/src/components/WelcomePage.tsx

```diff
@@ -2,54 +2,41 @@ import { Box, Flex, ListItem, Text, UnorderedList } from "@chakra-ui/react";
 import { useAsync } from "react-async-hook";
 import { Link } from "react-router-dom";
 import { getAllSongbooks } from "../services/songs";
+import SongbookIndexTable from "./SongbookIndexTable";
 
 export default function WelcomePage() {
   const asyncSongbooks = useAsync(async () => getAllSongbooks(), []);
   const songbooks = asyncSongbooks.result?.data.results;
   return (
     <Box>
       <Text
-        fontSize="2rem"
-        backgroundColor="black"
+        fontSize="3rem"
```

Comment:
Just a note for the future, no need to change: I *think* we should be using built-in chakra sizes here ("md", "lg", "xxl", etc) and probably also use `<Heading>` instead of text. That might be part of why this hasn't look good on mobile in the past. We can pretty this up for mobile later though.

Reply:
Noted the sizes. Heading is a good note as well; will integrate into a future PR.